### PR TITLE
Hotfix package

### PR DESCRIPTION
**Pull request by olegrog**

### CODE REVIEW

File: slm_porosity/heat_transfer.f90 [hunk 1]
```diff
@@ -987,16 +985,16 @@ ELEMENTAL FUNCTION lf_exponent (enthalpy, is_D)
     INTEGER,  INTENT(IN), OPTIONAL :: is_D
     REAL(pr) :: lf_exponent
 
-    lf_exponent = sigmoid(enthalpy, enthalpy_one, Dphi_one)
+    lf_exponent = sigmoid(enthalpy - enthalpy_one, Dphi_one, is_D)
   END FUNCTION lf_exponent
 
-  ELEMENTAL FUNCTION sigmoid(x, x0, slope, is_D)
```
Comment (shmidt9510) [inline slm_porosity/heat_transfer.f90:993]: x - x0 seems more intuitive

File: slm_porosity/heat_transfer.f90 [hunk 2]
```diff
@@ -718,7 +716,7 @@ SUBROUTINE check_user_input
     IF (.NOT.laser_inside_domain) &
       STOP '--- The initial laser position is out of domain ---'
     IF (dim.EQ.3 .AND. half_domain .AND. xyzlimits(1,2).NE.0.0_pr) &
-      STOP '--- half domain = true, the initial laser position does not located on the boundary ---'
+      STOP '--- The initial laser position is not on the plane y=0 ---'
```
Comment (shmidt9510) [inline slm_porosity/heat_transfer.f90]: Without a comment about halfdomain it source of the error is non understandable. y in [-5 0] and y in [0 5] are differs in this condition. But there nothing about it.

File: slm_porosity/heat_transfer.f90 (hunk 1 above)
Comment (olegrog) [reply slm_porosity/heat_transfer.f90:993]: "Seems" is not constructive argument. Just look at all standard functions. 
I will add `11) prefer the most laconic procedure/function signature`

File: slm_porosity/heat_transfer.f90 (hunk 2 above)
Comment (olegrog) [reply slm_porosity/heat_transfer.f90]: You are right. I tried to make comment more laconic.

**Review by shmidt9510 — APPROVED** [review]

**Pull request merged**